psi4/driver/driver.py

```diff
@@ -1602,6 +1602,14 @@ def hessian(name, **kwargs):
         wfn.set_gradient(G0)
 
         # Explicitly set the current energy..
+        if isinstance(lowername, str) and lowername in procedures['energy']:
+            # this correctly filters out cbs fn and "hf/cc-pvtz"
```

Comment:
Why is filtering out "hf/cc-pvtz" correct? I'd expect that to register as an HF TOTAL HESSIAN, HF TOTAL GRADIENT.

Reply:
`HF TOTAL HESSIAN` is what we'd like, but without the filtering, get `HF/CC-PVTZ TOTAL HESSIAN`

Reply:
It should be possible to strip the basisset at this point so we can still mark it as an HF hessian. Let's file an issue once this PR is in, so we don't lose track of this.

Reply:
I think the routing in DDD is different enough that this won't come up, but I'm adding it to the to-be-tidied list